pages/upload.php

```diff
@@ -0,0 +1,108 @@
+    returns true irrespective of authentication success or failure.
+    6. $Auth->getErrorTxt() - Returns errortxt if authentication process encounters a database bound issue otherwise return empty string irrespective
+    of authentication success or failure.
+
+    Relevant to Front-End:
+    1. $Uploader->getErrorTxt() - Returns error string which will be empty when no error is present.
+    2. $Auth->is_authenticated() - Returns the true if user is logged in, otherwise returns false.
+    3. $Auth->getFirstName() - Returns the first name of the logged in user, if a user is not logged in returns empty string.
+    4. $Auth->getLastName() - Returns the last name of the logged in user, if a user is not logged in returns empty string.
+    5. $Auth->getEmail() - Returns the email address of the logged in user, if a user is not logged in returns empty string
+    
+    Notes:
+    # Auth is used on this page to only allow a logged-in user to visit this page, if a non logged-in user stumbles across this page he/she will be
+    redirected to the login page instead.
+    # Refer to the samples provided below to incorporate the relevant functions.
+    # Front-End Team - Please use the same naming convention used for form button/field names as provided in the sample below.
+    # Front-End Team - DO NOT MAKE CHANGES TO ANY CODE OUTSIDE <html></html> TAGS. 
+
+*/
+
+// Connection & Authorization Process **STARTS**
+session_start();
+include("./helper/connect_class.php");
+$Connector = new Connectors;
+$db=$Connector->phptodbconnector();
+$Auth = new Auth;
+if(!$Auth->authenticate($_SESSION,$db)){
+    unset($_SESSION);
+}
+if(!$Auth->is_authenticated()){
+    header("Location: ./login.php");
+}
+// Connection & Authorization Process **ENDS**
+
+// File Upload Process **STARTS**
+
+include("./helper/uploader_class.php");
+$Uploader = new Uploader;
+if($Uploader->validateFormRequest($_POST,$_FILES)){
+    if($Uploader->createJob($db,$_SESSION,$_FILES)){
+        header("Location: ./download.php");
+    }
+}
+
+// File Upload Process **ENDS**
+?>
+<!DOCTYPE html>
+<html>
+<head>
+<title>Upload Page - Music2Score</title>
+<link rel="shortcut icon" type="image/jpg" href="images/favicon.ico"/>
+<link rel="stylesheet" href="plugins/bootstrap_v4.0/css/bootstrap.min.css">
+<link rel="stylesheet" href="plugins/fontawesome_v5.15.2/css/all.css">
+<link rel="stylesheet" href="css/components.css">
+<link rel="stylesheet" href="css/upload.css">
+</head>
+<body>
+<?php include("./components/navbar.php"); ?>
+<div class="body_content_container">
+<form action="./upload.php" method="POST" enctype="multipart/form-data">
+<div class="form_container">
+<div class="upload_container">
+<div class="design_container">
+<img class="large_image" src="./images/upload_image.jpg" style="width: 100%; padding-right: 30px;">
+</div>
+<div class="main_container">
+<h2>File Uploader</h2>
+<p>Easily convert any music midi file into music sheet.</p><br>

```

Comment:
Should prob say "sheet music" instead of "music sheet".

Reply:
No issues, you can change it while working on the progress bar element for the file uploader.